src/main/resources/site/parts/husleieCalculator/husleieCalculator.jsx

```diff
@@ -475,11 +493,13 @@ function HusleieCalculator(props) {
           </Row>
           <Row className='ms-0'>
             <Button className='submit-one-year' onClick={submitOneYearLater} ref={submitButton}>
-              {chooseFiguresToCalculateRent.oneYearLater.phraseOneYearLater}
+              {`${chooseFiguresToCalculateRent.oneYearLater.phraseOneYearLater}, ${buttonPhrase} eller`}
             </Button>
+            <span className='visually-hidden'>, knapp eller</span>
```

Comment:
Bruk variablen buttonPhrase, og opprett en variabel for "eller" slik at det blir riktig på engelsk.